ali-dbhub-client/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "importHelpers": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
For PRs that modify the documentation, dependencies should not be changed. If a change in dependencies is required, a separate PR should be opened for it and testing must be conducted.